client/src/components/application/Examine/CompletedCompName.vue

```diff
@@ -0,0 +1,308 @@
+
+      </div>
+
+    </div>
+
+    <!-- RESET COMMENT popup -->
+    <div class="modal fade" id="add-comment-reset-modal" tabindex="-1" role="dialog">
+      <div class="modal-dialog modal-lg" role="document">
+        <div class="modal-content">
+          <div class="modal-header">
+            <h5 class="modal-title">Please give a comment to explain why this NR is being RESET</h5>
+            <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+              <span aria-hidden="true">&times;</span>
+            </button>
+          </div>
+          <div class="modal-body">
+            <textarea id="reset-comment-text" class="form-control" rows="10"
+                      v-model="add_comment_display"></textarea>
+          </div>
+          <div class="modal-footer">
+            <button type="button" class="btn btn-sm btn-secondary"
+                    data-dismiss="modal" @click="cancelReset">Cancel</button>
+            <button type="button" id="reset-nr-after-comment-button" class="btn btn-sm btn-danger" disabled="true"
+                    data-dismiss="modal" @click="reset">RESET</button>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <!-- CANCEL COMMENT popup -->
+    <div class="modal fade" id="add-cancel-comment-modal" tabindex="-1" role="dialog">
+      <div class="modal-dialog modal-lg" role="document">
+        <div class="modal-content">
+          <div class="modal-header">
+            <h5 class="modal-title">Please give a comment to explain why this NR is being CANCELLED</h5>
+            <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+              <span aria-hidden="true">&times;</span>
+            </button>
+          </div>
+          <div class="modal-body">
+            <textarea id="cancel-comment-text" class="form-control" rows="10"
+                      v-model="cancel_comment_display"></textarea>
+          </div>
+          <div class="modal-footer">
+            <button type="button" class="btn btn-sm btn-secondary"
+                    data-dismiss="modal" @click="cancelNrCancel">Cancel</button>
+            <button type="button" id="cancel-nr-after-comment-button" class="btn btn-sm btn-danger" disabled="true"
+                    data-dismiss="modal" @click="cancelNr">CANCEL REQUEST</button>
+          </div>
+        </div>
+      </div>
+    </div>
+
+
+  </div>
+</template>
+
+<script>
+/* eslint-disable */
+  export default {
+    name: 'CompletedCompName',
+    data: function () {
+      return {
+        add_comment_display: "",
+        cancel_comment_display: "",
+        resetting: false,
+      }
+    },
+    computed: {
+      currentState() {
+        return this.$store.getters.currentState;
+      },
+      userId() {
+        return this.$store.getters.userId;
+      },
+      is_furnished() {
+        if (this.$store.getters.furnished === "Y") return true;
+        return false;
+      },
+      is_approved_expired() {
```

Comment:
I am not sure the significance of some of the states of these, I copied it from the CompName so I didn't lose anything, but without understanding the business purpose of them.